src/Microsoft.DotNet.XHarness.iOS.TestRunner/Microsoft.DotNet.XHarness.iOS.TestRunner.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Folder Include="Core\" />
+    <Folder Include="xUnit\" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Remove="xUnit\NUnit3Xml.xslt" />
+    <None Remove="xUnit\NUnitXml.xslt" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="xUnit\NUnit3Xml.xslt">
+      <XlfSourceFormat></XlfSourceFormat>
+      <XlfOutputItem></XlfOutputItem>
+    </EmbeddedResource>
+    <EmbeddedResource Include="xUnit\NUnitXml.xslt">
+      <XlfSourceFormat></XlfSourceFormat>
+      <XlfOutputItem></XlfOutputItem>
+    </EmbeddedResource>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="xunit.analyzers" Version="0.10.0" />
+    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="Mono.Options" Version="6.6.0.161" />

```

Comment:
Can we build a version without it?

Reply:
@marek-safar yes we can, we can take a look at how we are calling xharness, but AFAIK we do not use the cli for it.